Approved/implicitly-scoped-qubit-allocation.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Implicitly-scoped qubit allocation
+description: Provides more convenient syntax for allocating qubits.
+author: Sarah Marshall
+date: September 28, 2020
+---
+
+# Proposal
+
+1. Add new `use` and `borrow` statements to allocate qubits that are valid until the end of the current block.
+2. Add new scope statement that does not require a preceding keyword, and is executed exactly once; equivalent to `if (true) { }`.
+3. Deprecate and remove existing `using` and `borrowing` block statements.
```

Comment:
Not strictly related to this proposal, but would it be reasonable to offer a code action or other IDE support to help transition to the new proposed syntax?

Reply:
Yes, that sounds like a good idea.

Reply:
Added a note about this to Further Considerations.